packages/editor-ui/src/components/MainSidebar.vue

```diff
@@ -647,7 +647,7 @@ export default mixins(
 		height: 35px;
 		line-height: 35px;
 		color: $--custom-dialog-text-color;
-		--menu-item-hover-fill: var(--color-primary-tint-3);
+		--menu-item-hover-fill: var(--color-background-base);
```

Comment:
intentional change?

Reply:
Yes. Based on [this handoff note](https://www.notion.so/n8n/Handoff-26dfcbf10486499a98947777b8b10861#20701be8e268496889c8c5249d9e537d).